cmd/stupid/main.go

```diff
@@ -21,16 +22,20 @@ func main() {
 		}
 	}
 	if len(os.Args) != 3 {
-		fmt.Printf("Usage: stupid config.yaml 500\n")
-		os.Exit(1)
+		infra.ErrorExit(errors.New(infra.UsageError))
```

Comment:
you could create a new `Main` func that returns error, and calls it in actual `main`, and exit there. So we don't need `ErrorExit`